traceapp/tmpl/doc.go

```diff
@@ -3,4 +3,4 @@
 // This package needs to be go generated after making changes to template files.
 package tmpl
 
-//go:generate go run -tags=dev data_generate.go
+//go:generate vfsgen -tags=dev -source="sourcegraph.com/sourcegraph/appdash/traceapp/tmpl".Data -output-tags=!dev
```

Comment:
(You have a minor typo in your question, you meant to say "hard to read `-tags=dev` and `-output-tags=!dev` in the same line.")